st2common/bin/st2ctl

```diff
@@ -14,14 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-LOGFILE="/dev/null"
 COMPONENTS="st2actionrunner st2api st2stream st2auth st2garbagecollector st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2chatops mistral"
-STANCONF="/etc/st2/st2.conf"
+ST2_CONFIG="/etc/st2/st2.conf"
 
 # Ensure global environment is sourced if exists
 # Does not happen consistently with all OSes we support.
 [ -r /etc/environment ] && . /etc/environment
 
+# load in environment to allow override of COMPONENTS and ST2_CONFIG above
+# Ubuntu/Debian
+[ -r /etc/default/st2ctl ] && . /etc/default/st2ctl
```

Comment:
I'd like it better if we used `source` here just be be more explicit, but this shebang is bash so it shouldn't matter - not a blocker, just a preference.